shared/src/commonTest/kotlin/co/touchlab/kampkit/BreedRepositoryTest.kt

```diff
@@ -42,119 +34,112 @@ class BreedRepositoryTest {
     private val clock = ClockMock(Clock.System.now())
 
-    private var model: BreedRepository = BreedRepository(dbHelper, settings, ktorApi, kermit, clock)
+    private val repository: BreedRepository = BreedRepository(dbHelper, settings, ktorApi, kermit, clock)
 
     companion object {
-        private val appenzeller = Breed(1, "appenzeller", 0L)
-        private val australianNoLike = Breed(2, "australian", 0L)
-        private val australianLike = Breed(2, "australian", 1L)
-        val dataStateSuccessNoFavorite = DataState(
-            data = ItemDataSummary(appenzeller, listOf(appenzeller, australianNoLike))
-        )
-        private val dataStateSuccessFavorite = DataState(
-            data = ItemDataSummary(appenzeller, listOf(appenzeller, australianLike))
-        )
+        private val appenzeller = Breed(1, "appenzeller", false)
+        private val australianNoLike = Breed(2, "australian", false)
+        private val australianLike = Breed(2, "australian", true)
+        private val breedsNoFavorite = listOf(appenzeller, australianNoLike)
+        private val breedsFavorite = listOf(appenzeller, australianLike)
+        private val breedNames = breedsFavorite.map { it.name }
     }
 
-    @Test
-    fun staleDataCheckTest() = runTest {
-        val currentTimeMS = Clock.System.now().toEpochMilliseconds()
-        settings.putLong(BreedRepository.DB_TIMESTAMP_KEY, currentTimeMS)
-        assertTrue(ktorApi.calledCount == 0)
-
-        val expectedError = DataState<ItemDataSummary>(exception = "Unable to download breed list")
-        val actualError = model.getBreedsFromNetwork(0L)
-
-        assertEquals(
-            expectedError,
-            actualError
-        )
-        assertTrue(ktorApi.calledCount == 0)
+    @AfterTest
+    fun tearDown() = runTest {
+        testDbConnection.close()
     }
 
     @Test
-    fun updateFavoriteTest() = runBlocking {
+    fun `Get breeds without cache`() = runBlocking {
         ktorApi.prepareResult(ktorApi.successResult())
-
-        flowOf(model.refreshBreedsIfStale(), model.getBreedsFromCache())
-            .flattenMerge().distinctUntilChanged().test {
-                // Loading
-                assertEquals(DataState(loading = true), awaitItem())
-                // No Favorites
-                assertEquals(dataStateSuccessNoFavorite, awaitItem())
-                // Add 1 favorite breed
-                model.updateBreedFavorite(australianNoLike)
-                // Get the new result with 1 breed favorited
-                assertEquals(dataStateSuccessFavorite, awaitItem())
-            }
+        repository.refreshBreedsIfStale()
+        repository.getBreeds().test {
+            assertEquals(breedsNoFavorite, awaitItem())
+        }
     }
 
     @Test
-    fun fetchBreedsFromNetworkPreserveFavorites() = runBlocking {
-        ktorApi.prepareResult(ktorApi.successResult())
+    fun `Get updated breeds with cache and preserve favorites`() = runBlocking {
```

Comment:
For this project it's doesn't matter, but these sentence names don't work for JS. I personally prefer the function names anyway. I wouldn't change this, but I feel like this goes in the "best practice" discussion...